### DISCUSSION TITLE
[Unstable] using toolbelt while healing menu is open does not work

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15416] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/56ea681f-947e-43de-ab08-6a060c58636a)
the slots that overlap with the health UI cant be selected

### Reproduction steps

1. try healing with any of top 4 slots in toolbelt
2. cry

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### -

new unstable with the alien ruin pvp map

### Version

Other

### Which operating system did you encounter this bug on?

Windows

**Comment by EebleSheembl** [comment]
I've found that if you click on them on the very left side it uses them

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming up in the next unstable build.
